tools/mksurfdata_esmf/README

```diff
@@ -76,6 +78,7 @@ slevis HAS RUN THESE CASES and HAS LISTED ISSUES ENCOUNTERED
 ------------------------------------------------------------
 REMEMBER TO compare against existing fsurdat files in
 /glade/p/cesmdata/cseg/inputdata/lnd/clm2/surfdata_map/release-clm5.0.18
+0) New 30-sec raw data for soil texture fails. Try requesting more mem.
```

Comment:
In this README file, this is the only substantial change (others just improve the formatting):
It documents that I haven't run successfully with the new 30-sec soil-texture data.